packages/donations-app/app/components/project/projectControllerV2.js

```diff
@@ -15,11 +15,18 @@ angular.module('aliceApp')
 				if(impact) {
 					elem.totalPercentage = Math.floor(100 * impact.totalValidatedForOutcome / elem.amount);
 					elem.totalValidatedForOutcome = impact.totalValidatedForOutcome;
-				} else { elem.totalPercentage = 0; }
+				} else {
+					elem.totalPercentage = 0;
+					elem.totalValidatedForOutcome = 0;
+				}
 				elem.lightColor = elem.color ? convertHex(elem.color, 0.4) : 'rgba(255, 255, 255, 0.3)';
 			});
 
 			vm.model.percentageCompleted = Math.floor((100 * vm.model.amountValidated) / vm.model.fundingTarget);
+
+			// Split myStory array into 2 arrays: 1. of length 3, 2. of length: myStory.length - 3;
```

Comment:
Is it my story??
What if we have more than 6 outcomes, anyway it's not very elegant, we can do it in ng-repeat using the
https://docs.angularjs.org/api/ng/filter/limitTo

Reply:
limitTo was considered but thought this would be better to not clutter html because uibcollapse was used - reverted now :)